components/director/pkg/graphql/schema.graphql

```diff
@@ -197,6 +204,11 @@ enum RuntimeStatusCondition {
 	FAILED
 }
 
+enum ScheduledOperationType {
```

Comment:
maybe just `OperationType`, or even `Type` instead of `ScheduledOperationType` is enough?